src/Rings/mpoly-ideals.jl

```diff
@@ -2003,3 +2003,72 @@ function Base.show(io::IO, I::MPolyIdeal)
     print(io, "Ideal ", _get_generators_string_one_line(I))
   end
 end
+
+
+_pluecker_sgn(a::Vector{Int}, b::Vector{Int}, t::Int)::Int =
+  (-1)^(count(z -> z > t, a) + count(z -> z < t, b))
+
+@doc raw"""
+    flag_pluecker_ideal(dimensions::Vector{Int},n::Int,F)
+
+Returns generators of the defining ideal for the flag variety
+$\text{Fl}((d_1,\dots,d_k),n,\mathbb{F})$, where $(d_1,\dots,d_k)
+=$`dimensions` denotes the rank, with $d_j\leq n-1$.  That is, the vanishing
+set of this ideal corresponds to the space of $k$-step flags of linear
+subspaces $V_1\subset\dots\subset V_k$ in $\mathbb{F}^n$, where
+$\text{dim}(V_j) = d_{j}$.  You can obtain the generators for the
+\emph{complete flag variety}$ of $\mathbb{F}^{n}$ by taking `dimensions`
+$=(1,\dots,n-1)$ and `n`$=n$.  We remark that evaluating for `F = QQ` yields
+the same set of generators as any field of characteristic $0$.
+
+# Example
+Complete flag variety $\text{Fl}((1,2,3),4,\mathbb{Q})$. 
+```jldoctest
+julia> flag_pluecker_ideal([1,2,3],4,QQ)
+Ideal generated by
+  x[[1]]*x[[3, 4]] - x[[3]]*x[[1, 4]] + x[[4]]*x[[1, 3]]
+  -x[[1, 4]]*x[[2, 3]] + x[[2, 4]]*x[[1, 3]] - x[[3, 4]]*x[[1, 2]]
+  x[[2]]*x[[3, 4]] - x[[3]]*x[[2, 4]] + x[[4]]*x[[2, 3]]
+  x[[1]]*x[[2, 4]] - x[[2]]*x[[1, 4]] + x[[4]]*x[[1, 2]]
+  x[[1]]*x[[2, 3]] - x[[2]]*x[[1, 3]] + x[[3]]*x[[1, 2]]
+  x[[1]]*x[[2, 3, 4]] - x[[2]]*x[[1, 3, 4]] + x[[3]]*x[[1, 2, 4]] - x[[4]]*x[[1, 2, 3]]
+  -x[[1, 4]]*x[[1, 2, 3]] + x[[1, 3]]*x[[1, 2, 4]] - x[[1, 2]]*x[[1, 3, 4]]
+  -x[[2, 4]]*x[[1, 2, 3]] + x[[2, 3]]*x[[1, 2, 4]] - x[[1, 2]]*x[[2, 3, 4]]
+  -x[[3, 4]]*x[[1, 2, 3]] - x[[1, 3]]*x[[2, 3, 4]] + x[[2, 3]]*x[[1, 3, 4]]
+  -x[[1, 4]]*x[[2, 3, 4]] + x[[2, 4]]*x[[1, 3, 4]] - x[[3, 4]]*x[[1, 2, 4]]
+
+```
+# Example 2
+
+Flag variety $\text{Fl}((1,3),4,QQ)$.
+```jldoctest
+julia> flag_pluecker_ideal([1,3],4)
+Ideal generated by
+  x[[1]]*x[[2, 3, 4]] - x[[2]]*x[[1, 3, 4]] + x[[3]]*x[[1, 2, 4]] - x[[4]]*x[[1, 2, 3]]
+```
+"""
+function flag_pluecker_ideal(dimensions::Vector{Int}, n::Int, F::Field)
```

Comment:
```suggestion
function flag_pluecker_ideal(F::Field, dimensions::Vector{Int}, n::Int)
```
i think most similar functions take the field as the first argument